pkg/cmd/step/helm/step_helm_apply.go

```diff
@@ -99,73 +100,60 @@ func NewCmdStepHelmApply(commonOpts *opts.CommonOptions) *cobra.Command {
 	return cmd
 }
 
-func (o *StepHelmApplyOptions) Run() error {
-	var err error
+// Run runs on step helm apply
+func (o *StepHelmApplyOptions) Run() (err error) {
+
 	chartName := o.Dir
 	dir := o.Dir
 	releaseName := o.ReleaseName
 
 	// let allow arguments to be passed in like for `helm install releaseName dir`
 	args := o.Args
-	if releaseName == "" && len(args) > 0 {
-		releaseName = args[0]
-	}
 	if dir == "" && len(args) > 1 {
 		dir = args[1]
 	}
 
 	if dir == "" {
 		dir, err = os.Getwd()
 		if err != nil {
-			return err
+			return
```

Comment:
We should wrap here the error message instead of just returning the implicit error. You can use the `errors.Wrap(err, "get current working directory")` to add some context to the error.